Dfe.Data.Common.Infrastructure.CognitiveSearch/SearchByKeyword/Providers/SearchRuleProvider.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Dfe.Data.Common.Infrastructure.CognitiveSearch.SearchByKeyword.Options;
+
+namespace Dfe.Data.Common.Infrastructure.CognitiveSearch.SearchByKeyword.Providers;
+
+/// <summary>
+/// Facilitates search rules to be specified when running a search
+/// </summary>
+public class SearchRuleProvider : ISearchRuleProvider
+{
+    private SearchRuleOptions _ruleOptions;
+
+    /// <summary>
+    /// Construct the saerch rules provider, injecting into it the <see cref="SearchRuleOptions"/> to be applied
+    /// </summary>
+    /// <param name="searchRuleOptions"></param>
+    public SearchRuleProvider(SearchRuleOptions searchRuleOptions)
```

Comment:
Just thinking, why are we even bothering with options if we only have a single hard-coded rule???

Reply:
So we can opt to not use the rule without changing the code